src/OpenTelemetry.Instrumentation.AspNet/README.md

```diff
@@ -126,7 +126,7 @@ Currently, the instrumentation supports the following metric.
 
 | Name  | Instrument Type | Unit | Description |
 |-------|-----------------|------|-------------|
-| `http.server.duration` | Histogram | `ms` | Measures the duration of inbound HTTP requests. |
+| `http.server.request.duration` | Histogram | `s` | Measures the duration of inbound HTTP requests. |
```

Comment:
Change the desc as well: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration

Reply:
Previously, I have taken it from the code. Fixed documentation and the code in 6444a08d53a7dd13a6568f884461ae252b643ef7. Both for OWIN and ASP.NET.